packages/playwright-core/src/server/injected/highlight.ts

```diff
@@ -91,6 +92,11 @@ export class Highlight {
 
   install() {
     this._injectedScript.document.documentElement.appendChild(this._glassPaneElement);
+
+    this._glassPaneInterval = setInterval(() => {
```

Comment:
I've tried using `MutationObserver` as the more canonical solution over `setInterval`, but couldn't get that to work.